builtin/game/chat.lua

```diff
@@ -1053,24 +1053,58 @@ core.register_chatcommand("days", {
 	end
 })
 
+local function parse_shutdown_param(param)
```

Comment:
I assume this will go away with the squash?

Reply:
IDK, I guess so. We didn't touch this file.